### DISCUSSION TITLE
Landscape page designed for a wide table turned into a vertical page when cross reference the table

### DISCUSSION THREAD

**Discussion by ellaninomiya** [quarto-dev/quarto-cli#11399] [Q&A] [answered]
### Description

Hi, I am using kableExtra package to produce latex table in PDF format. One table is too wide so I have to place it in a landscape page, it worked fine, but when I cross reference this table, the `landscape` option didn't work and landscape page turned into a vertical page. How can I cross reference a wide table and still keep a landscape page? Thanks!

no cross reference, landscape page

```
library(kableExtra)
tb <- data.frame(rbind(
  seq(0, 1, 0.1), seq(1, 2, 0.1), seq(2, 3, 0.1), seq(3, 4, 0.1)))
kbl(tb, booktabs = T, escape = F) %>%
  landscape()
```

cross reference, landscape page turned into a vertical page. See @tbl-ls

```
#| label: tbl-ls
#| eval: true
#| echo: false
#| warning: false
#| tbl-cap: "landscape"

kbl(tb, booktabs = T, escape = F) %>%
  landscape()
```

**Comment by mcanouil** [comment, marked as answer]
Don't use third party to handle LaTeX as those aren't aware of the custom/advanced cross-reference system set in place in Quarto.

Either write the LaTeX command inside a raw code block (https://quarto.org/docs/authoring/markdown-basics.html#raw-content) or the new (Quarto 1.6) fenced div: https://prerelease.quarto.org/docs/authoring/article-layout.html#landscape-mode

**Reply by ellaninomiya** [reply]
Thanks! The new fenced div in Quarto 1.6 is really helpful.
